coldfront/config/local_settings.py.sample

```diff
@@ -264,6 +264,22 @@ LOCAL_SETTINGS_EXPORT += [
 # ColdFront Plugin Settings
 #===============================================================================
 
+#------------------------------------------------------------------------------
```

Comment:
Can you move this section after "Enable XDMoD support"?

Reply:
(The reason being: that section is the last one that came out of the box with Coldfront; everything after was added by us.)